src/NServiceBus.TransactionalSession/TransactionalSessionBase.cs

```diff
@@ -50,7 +50,7 @@ public virtual Task Open(OpenSessionOptions options = null, CancellationToken ca
             }
 
             this.options = options ??= new OpenSessionOptions();
-            SessionId = options.CustomSessionId ?? Guid.NewGuid().ToString();
+            SessionId = options.SessionId ?? Guid.NewGuid().ToString();
```

Comment:
that isn't really necessary anymore since SessionId should always be set I think?